packages/grid/_modules_/grid/components/cell/GridCell.tsx

```diff
@@ -59,6 +60,7 @@ export const GridCell = React.memo((props: GridCellProps) => {
     tabIndex,
     value,
     width,
+    className,
```

Comment:
sort ASC

x the other cases

Reply:
I've updated the interfaces with the definitions of the props.